### ISSUE TITLE
[outreach] Surprised to find that among the direct children of Biological Process is "presynaptic process involved in chemical synaptic transmission".

### ISSUE THREAD

**Issue by cmungall** [geneontology/go-ontology#15235]
See:

https://twitter.com/RyanGutenkunst/status/966439663861837824

This comment is correct; it is a direct is-a child of BP. However, it has more specific parentage via part-of. E.g.

![image](https://user-images.githubusercontent.com/50745/36519673-aaeda30e-1741-11e8-9d72-d334132b85fd.png)

Some displays make this less obvious:

![image](https://user-images.githubusercontent.com/50745/36519709-e2146b2e-1741-11e8-98c4-eef8154d82be.png)

in fact if you look in displays like Protege the is-a is even more prominent

A few things we need to do here

1. BP is too broad an is-a parent. This is clearly a cellular process. This is also very broad, but it's slightly better.  [ontology group]

2. Displays need to be improved universally. Majority of users don't care about is-a vs part-of. Default should be to do the transitive reduction on the combined graph prior to display [informatics]

3. Insofar as we do prominently display is-a and part-of we should have better docs on why someone should care and explaining that sometimes direct is-a parent may be quite general [documentation]

**Comment by pgaudet** [comment]
I don't think 'cellular process' would convince user that we've address the issue. How about 'presynaptic process'? (well what is a 'presynaptic process'...?) 

Thanks, Pascale

**Comment by BarbaraCzub** [comment]
I agree we could add 'presynaptic process' (Potential definition: "Any process that is carried out at a presynapse."). But I do not think it should be as a child of 'cellular process'. If we add 'presynaptic process', we should also add 'postsynaptic process' (and make 'GO:0098926 postsynaptic signal transduction' a part_of child of that). Additionally, we should add 'synaptic process' as an is_a child of 'cellular process' and an is_a parent of both 'pre-' and 'postsynaptic process' terms; this could be defined as: "Any process that is carried out at a synapse, and not necessarily restricted to a single cell." (Based on: 'GO:0009987 cellular process' definition: 'Any process that is carried out at the cellular level, but not necessarily restricted to a single cell. For example, cell communication occurs among more than one cell, but occurs at the cellular level.').

**Comment by pgaudet** [comment]
But isn't "Any process that is carried out at a presynapse." = CC presynapse ? 

and therefore "presynaptic process involved in chemical synaptic transmission"
= BP chemical synaptic transmission + CC presynpase? 

I would be in favor of a merge here... 

Pascale

**Comment by deustp01** [comment]
The user has hit on a general problem: granularity at the top level is irregular. The specific patch is fine, but it is only a patch when what's needed is a systematic review, reorganization of existing terms, and creation of new ones to give more uniform coverage (and of course the resources, both internally and from the community to do the work).

At the same time, it does highlight a problem, and raise a practical issue - should some resources be diverted from once again re-working the distinctions among kinds of receptors and kinds of signaling associated with them (or not) - an important but specialized problem - to improving the high-level terms that are key tools for many users, as Jim Hu points out.

**Comment by BarbaraCzub** [comment]
@pgaudet 

I am not sure what you mean. I agree that if we add the term 'presynaptic process' it should be an occurs_in child of CC presynapse.

Wrt "and therefore "presynaptic process involved in chemical synaptic transmission"
= BP chemical synaptic transmission + CC presynpase?" - do you mean that "presynaptic process involved in chemical synaptic transmission" should be and is_a child of "chemical synaptic transmission" rather than a "part_of" child of it?

What exactly are you thinking of merging with what?

**Comment by pgaudet** [comment]
Hi @BarbaraCzub , 

Sorry that wasn't very clear. What I was trying to say is that if, as you suggest, presynaptic process' is "Any process that is carried out at a presynapse.", then its not different from saying that there is an unknown biological process occuring in a presynapse, which is the same as simply making a CC = presynapse annotation. 

If we agree on that first part, then the corollary is that  "presynaptic process involved in chemical synaptic transmission" should be annotated as BP= "chemical synaptic transmission"  and CC= presynapse. 

Anyway the term "chemical synaptic transmission"" starts with "The vesicular release of classical neurotransmitter molecules from a **presynapse**". 

What I was proposing to merge is  "presynaptic process involved in chemical synaptic transmission" , into "chemical synaptic transmission". 

Thanks, Pascale

**Comment by BarbaraCzub** [comment]
"presynaptic process involved in chemical synaptic transmission" is a term from the Synapse slim, so we should check with the Synapse project team. @Pimmelorus do you agree with this suggested merge?

cc @RLovering @dosumis

**Comment by cmungall** [comment]
Minimally this term should have a 'do_not_annotate' tag, it appears to be there for grouping purposes. I will leave it to the experts to determine the utility of the class (when we get the answer we should add it as metadata on the class).

Note that if we do decide it is not useful, a merge to 'chemical synaptic transmission' is not appropriate. It would be obsolete+consider. This is because there are currently children that are postsynaptic.

**Comment by dosumis** [comment]
This is intended as a grouping term.   I neglected to add a do-not-annotate tag (thought I had).  Agree it should have that as a minimum and that a more specific parent class (at least cellular process) would have been better.  

TL;DR - I agree that this term may be too general, but if you do get rid of it, I think it's worth thinking about what term or terms are most useful for grouping presynaptic processes that are fundamental to  function of the pre-synapse in synaptic transmission.   From what I understand of this community, this would be very useful. 

Some background in case you're interested: The synapse curators at VU were insistent on having grouping classes for both pre- and post-synaptic processes at the synapse that are always or sometimes part of (or regulate / prepare the presynapse for?) synaptic transmission.  Synaptic transmission is a classified under cell-cell signalling. This encompasses release of a signal (the final step of the presynaptic part of synaptic transmission), reception and downstream signalling pathways.  This partonomy comfortably fits in the postynaptic part of synaptic transmission.  It less comfortably fits the presynaptic part.  Do we include all the upstream processes leading to release of the signal?  This doesn't fit well with cell-cell signaling in general. Do we include the many presynaptic processes that play a part in all cases of synaptic transmission but that also occur outside of that context.   What about the synaptic vesicle cycle - that prepares the presynapse for signal release and which VU also wanted grouped?

Based on these considerations and VU's strong desire for a term. I added the general class we're discussing here. I was slightly queasy about adding such a general class at the time.  On reflection, maybe it's not really necessary.  I added extensive 'occurs in' links which will serve to group annotations to the processes to presynapse if occurs_in is used in enrichment.  I think it would be defensible to be a bit less rigid with the partonomy of synaptic transmission in the presynapse (a small amount of false grouping on partonomy here might be better than lack of grouping/enrichment expected by biologists.  When I built this I was pushing hard for curators to add 'part of' 'chemical synaptic transmission' as an extension where relevant to make up for this conservative partonomy in the ontology, but this seems to have been dropped with their move to simpler, expert-driven curation.

**Comment by dosumis** [comment]
Peter wrote: 

> The user has hit on a general problem: granularity at the top level is irregular. The specific patch is fine, but it is only a patch when what's needed is a systematic review, reorganization of existing terms, and creation of new ones to give more uniform coverage (and of course the resources, both internally and from the community to do the work).

> At the same time, it does highlight a problem, and raise a practical issue - should some resources be diverted .. to improving the high-level terms that are key tools for many users, as Jim Hu points out.

This ticket may not be the best place to discuss this (probably better on email thread), but can you outline how "higher level terms" that give "more uniform coverage" are "key tools" for many users?  Do you know of any enrichment software or analysis that bakes in some semantics of levels or depth in GO?  Is this an actual problek for analysis or a potential irritation for those browsing in tools like OLS, AmiGO or QuickGO? How many levels down should regular (parallel?) granularity be maintained too?  How can we judge this?

**Comment by deustp01** [comment]
Hi David,
I was picking up on something Val (I think) said on the twitter thread earlier today, to the effect that biologists new to GO have been known to start near the tops of ontologies to find a sensible brod organization of domains of knowledge. Indeed, the original unhappy tweet took exactly this form: “Why is [specific small topic] presented at the same high level as [canonical broad topics]?”
Maybe GO doesn’t need to do business with that class of users? If so, I withdraw the comment and the suggestion.
Peter

**Comment by pgaudet** [comment]
Shouldn't all these terms somehow be children of 'nervous system process'? Starting at "synaptic signaling" (which could perhaps also be the parent of 'presynaptic process involved in chemical synaptic transmission')? 


Thanks, Pascale

**Comment by pgaudet** [comment]
@BarbaraCzub Do you have any feedback on my last comment ?

**Comment by BarbaraCzub** [comment]
Hi @pgaudet I agree that these terms should be related to the 'nervous system process'.

**Comment by pgaudet** [comment]
@BarbaraCzub Thanks - do you want to add the subclasses, or would you like me to do it ? 

Thanks, Pascale

**Comment by BarbaraCzub** [comment]
Could you please go ahead and add these @pgaudet ?

Thanks,
Barbara

**Comment by pgaudet** [comment]
Hi @BarbaraCzub 
I've added is_a GO:0050877 ! nervous system process to both 
- presynaptic process involved in chemical synaptic transmission
and 
- chemical synaptic transmission, postsynaptic

And added 'do not annotate' tags. 

Let me know if I can merge. 

Thanks, Pascale

**Comment by BarbaraCzub** [comment]
Thanks @pgaudet, yes, please merge.
